starknet_devnet/blueprints/base.py

```diff
@@ -32,6 +32,18 @@ def dump():
     state.dumper.dump(dump_path)
     return Response(status=200)
 
+@base.route("/load", methods=["POST"])
+def load():
+    """Loads the starknet_wrapper"""
+
+    request_dict = request.json or {}
+    load_path = request_dict.get("path") or state.dumper.dump_path
```

Comment:
We probably don't want `state.dumper.dump_path` to be used for loading, I think the `or` and its right hand side should be removed.